docs/source/transforms.rst

```diff
@@ -51,12 +51,37 @@ Make sure to use only scriptable transformations, i.e. that work with ``torch.Te
 
 For any custom transformations to be used with ``torch.jit.script``, they should be derived from ``torch.nn.Module``.
 
+In TorchVision we implemented 3 policies learned on the following datasets: ImageNet, CIFA10 and SVHN.
 
 Compositions of transforms
 --------------------------
 
 .. autoclass:: Compose
 
+
+AutoAugment transforms
+----------------------
+
+`AutoAugment <https://arxiv.org/pdf/1805.09501.pdf>`_ is a common Data Augmentation technique that can improve the accuracy of Scene Classification models.
```

Comment:
instead of mentioning `scene classification`, can you mention instead `image classification`?